### DISCUSSION TITLE
What is the meaning of "A, U, M, P, D" letters in commit messages?

### DISCUSSION THREAD

**Discussion by sergioro9** [easylist/easylist#17371] [Q&A]
When I run `git log` I can see commit messages like `M: https://www.miniminiplus.pl/`. I wonder what the initial letter means. My guess:

```
A: Add 
M: Modify 
U: Update
P: ???
D: Delete
```

**Comment by iam-py-test** [comment]
Does this answer your question: https://github.com/easylist/easylist/discussions/16755 
Thanks

**Reply by sergioro9** [reply]
#16755 explains `P:` but it does not explain `U:`. Thanks anyways.

**Reply by iam-py-test** [reply]
Where have you seen `U:` in use? I do not see any recent commits using it.
Thanks

**Reply by sergioro9** [reply]
The `U:` is in various commit messages including:

158374208f8b609139e14d05e6d18d43047d4b9c
c891ba7980085990c4cf774df315af962fb8d765
f316bc037d570532987ce6964b579b52fcaf9861

now I'm pretty sure it means "Update".

**Reply by iam-py-test** [reply]
Interesting.

**Comment by Yuki2718** [comment]
The question here is answered by the priveious discussion https://github.com/easylist/easylist/discussions/16755 Those `U:` or `D:` are not official and used by PR-er (though ABP officially uses `D:` in their repo). EL maintainers use `FOP.py` which accepts only `A:`, `P:`, or  `M:`.
